protocols/src/N2NP.nroff

```diff
@@ -235,17 +235,17 @@ It is described with the value 0x2 (2) in the packet type field.
 SERVICE
 .LP
 .in 2
-The SERVICE packet is used to describe a payload type, in order to the
+The SERVICE packet is used to describe a payload type in order for the
 receiver to know what type of data will follow the header. If a SERVICE type
-is specified, data payload MUST follow the header. A data payload shall be
-always in the following frame:
+is specified a data payload MUST follow the header. A data payload shall
```

Comment:
`If a SERVICE type is specified [...]` is a complement, and since it's at the start of the sentence, it does requires to be separated from the rest of it with a comma.
Like this:
```
If a SERVICE type is specified, a data payload MUST follow the header.
                              ^ 
```

Reply:
Yes, I have no clue why I removed it...